bin/overrides.sh.erb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+requote() {
+  local res=""
+  for x in "${@}" ; do
+    grep -q "[[:space:]]" <<< "$x" && res="${res} '${x}'" || res="${res} ${x}"
+  done
+  sed -e 's/^ //' <<< "${res}"
+}
+
+function <%=command%> {
+  local arguments=$(requote "${@}")
+  bash -c "<%=command_path%> ${arguments}"
+}
+
+readonly -f <%=command%> &> /dev/null
```

Comment:
This is really what makes this work, as it keeps the function from being overridden by the genuine one.